netci.groovy

```diff
@@ -0,0 +1,32 @@
+import jobs.generation.Utilities
+
+def project = GithubProject
+def branch = GithubBranchName
+def isPR = true
+def platformList = ['Windows_2016:WindowsServerCore-ltsc2016', 'Windows_2016:WindowsServerCore-1709']
+def versionList = ['3.5', '4.6.2', '4.7.1']
```

Comment:
Each time a new version is added/removed, one has to edit this list. In a way it is good to revisit the coverage but there is maintenance cost.

Reply:
Just realized there is an issue here - 3.5 and 4.7.1 are the only versions supported on 1709.  You can see it in https://github.com/Microsoft/dotnet-framework-docker/blob/master/build-pipeline/pipeline.json.  CI is going to have to account for this.

Reply:
Good point! I will update accordingly.